rzk/src/Rzk/Main.hs

```diff
@@ -69,6 +70,7 @@ parseRzkFilesOrStdin = \case
       then do
         putStrLn ("Using Rzk project stucture specified in " <> rzkYamlPath)
         paths <- extractFilesFromRzkYaml rzkYamlPath
+        when (null paths) (error $ "No files found in " <> rzkYamlPath)
```

Comment:
This is a weird error message to show when `include` is empty.
This seems to be the error message about missing `rzk.yaml`?

Reply:
I simply copied the same error from [`expandRzkPathsOrYaml`](https://github.com/rzk-lang/rzk/blob/dfa2a70454df25e9bf039d82eeb2d6e602ecf92b/rzk/src/Rzk/Main.hs#L57). I didn't interpret it as meaning `rzk.yaml` itself is missing, but rather that there are no files listed inside it.

Reply:
I see, still I think we should do better.

Reply:
Ok, what error message would you prefer?